managed-databases/document-database/how-to/manage-snapshots.mdx

```diff
@@ -0,0 +1,84 @@
+---
+meta:
+  title: How to manage Document Database snapshots
+  description: This page explains how to manage database snapshots
+content:
+  h1: How to manage Document Database snapshots
+  paragraph: This page explains how to manage database snapshots
+tags: managed-database database database-snapshot ferretdb mongodb document
+dates:
+  validation: 2023-08-01
+  posted: 2023-08-01
+categories:
+  - managed-databases
+---
+
+[Snapshots](/managed-databases/document-database/how-to/manage-snapshots/) allow you to restore data in a new Database Instance.
+
+When you set up [Autobackups](/managed-databases/document-database/how-to/enable-autobackup/) the automatic backups are created in Snapshot format.
```

Comment:
I'm not sure if snapshot should be capitalized or not capitalized? Or is it correct that in certain contexts it is, and others not?

Reply:
```suggestion
When you set up [Autobackups](/managed-databases/document-database/how-to/enable-autobackup/) the automatic backups are created in snapshot format.
```